validator/sawtooth_validator/consensus/poet1/validator_registry.py

```diff
@@ -293,7 +411,11 @@ def __init__(self, minfo=None):
         self.update = None
 
         if 'Update' in minfo:
-            self.update = Update(minfo=minfo['Update'])
+            if 'signup_info' in minfo['Update']:
```

Comment:
@dcmiddle So, I'll implement the ValidatorRegistryTransaction as the subclass of the [UpdateTransaction](https://github.com/hyperledger/sawtooth-core/blob/master/validator/journal/transaction.py#L228). What's your point?